docs/plugins/videoplayer.md

```diff
@@ -37,30 +38,31 @@ class Player extends Lightning.Component {
 }
 ```
 
-In the `_firstActive` (or `_init`) lifecycle event, you can pass the reference to the component that should be set as the _consumer_.
+In the `_firstActive` (or `_init`) [lifecycle event](../../lightning-core-reference/Components/LifecycleEvents.md), you can pass a reference to the component that should be set as the *consumer*.
 
-Note that only one component at the same time can consume VideoPlayer events.
+> Only *one* component can consume VideoPlayer events at the same time.
 
 ### position
 
 Sets the x and y position of the video player.
 
-The `position` method accepts 2 arguments (`top` and `left`). Both values should be absolute numbers (either positive or negative).
-The default values for `top` and `left` are `0`.
+The `position` method accepts 2 arguments: `top` and `left`. Both values must be positive or negative absolute numbers. They both default to 0 (zero).
+
+For example:
```

Comment:
I don't think we need 'for example' for each code example :)

Reply:
Okay!